README.md

```diff
@@ -2,6 +2,7 @@
 
 [![PyPI](https://img.shields.io/pypi/v/stock-indicators?color=blue&label=PyPI)](https://badge.fury.io/py/stock-indicators)
 [![code coverage](https://img.shields.io/azure-devops/coverage/skender/stock.indicators/26/main?logo=AzureDevOps&label=Test%20Coverage)](https://dev.azure.com/skender/Stock.Indicators/_build/latest?definitionId=26&branchName=main&view=codecoverage-tab)
```

Comment:
Not needed for this PR, but after we've figured out how to get the [coverage badge from Codacy](https://github.com/DaveSkender/Stock.Indicators.Python/pull/336), we'll need to change this from the obsolete Azure DevOps source.

Reply:
Right. We should replace. Are you having a problem with that task? Or just doing now? (Just a question😅)

Reply:
I started it, but for some reason the badge didn't work, so I stopped working on it.  Feel free to try it out.  Maybe you'll have better luck.  No need to hold up this PR though.

Reply:
Okay, Please add me Codacy project and TestPypi.